docs/workflow_tasks/lzards_backup.md

```diff
@@ -0,0 +1,43 @@
+---
+id: lzards_backup
+title: LZARDS Backup
+hide_title: false
+---
+
+The LZARDS backup task takes an array of granules and initiates backup requests to the LZARDS API, which will be handled asynchronously by LZARDS.
+
+## Deployment
+
+The LZARDS backup task is not automatically deployed with Cumulus. Rather, in order to deploy the task through the Cumulus module, you must specify a `lzards_launchpad_passphrase` in your terraform variables. Additionally, ensure that you are passing that variable by ensuring that the Cumulus module has the following input value configuration:
+
+```
+lzards_launchpad_passphrase  = var.<launchpad_passphrase_variable_name>
+```
+
+In short, deploying the LZARDS task requires configuring a passphrase and ensuring that your TF configuration passes that variable into the Cumulus module.
+
+Additional terraform configuration for the LZARDS task can be found in the Cumulus `ingest` module's `variables.tf`, where the the relevant variables are prefixed with `lzards_`.
```

Comment:
```suggestion
Additional terraform configuration for the LZARDS task can be found in the [`cumulus` module's `variables.tf` file,](https://github.com/nasa/cumulus/blob/master/tf-modules/cumulus/variables.tf) where the the relevant variables are prefixed with `lzards_`. You can add these variables to your deployment using the same process outlined above for `lzards_launchpad_passphrase`. 
```

They should be referring to the values that the `cumulus` module supports, not `ingest` I think because not all values supported by `ingest` may be supported by `cumulus`. Plus, the only module they're using directly is `cumulus`